src/static/css/page.css

```diff
@@ -942,6 +942,13 @@ pre {
   left: 0;
 }
 
+@media (max-width: 56.25em) {
+  .webmention-replies-item .webmention-author .webmention-author-avatar,
+  .webmention-mentions-item .webmention-author .webmention-author-avatar {
+    position: static;
+  }
+}
```

Comment:
You can actually increase specificity of https://github.com/HTTPArchive/almanac.httparchive.org/pull/2667/files#diff-bde4749c6cd5fcbfc940dd0526fa7de34c53aed8bcb972a726311fc70ec7353bR930-R936 and that should bring the exact same look as the desktop (if that's what you want). Else this look is also fine and utilizes the space better I feel.

Reply:
That link isn't working for me.

Reply:
This is a better link I guess since github not opening lines outside of the changed area nicely: https://github.com/HTTPArchive/almanac.httparchive.org/blob/2f0bea041a4105ce433203c65a46aebefe126926/src/static/css/page.css#L930-L936